apps/root/src/common/utils/transaction-history/parsing.tsx

```diff
@@ -598,7 +637,6 @@ const parseEarnClaimDelayedWithdrawApiEvent: ParseFunction<
       recipient: event.data.recipient,
       status: TransactionStatus.DONE,
       tokenFlow: TransactionEventIncomingTypes.INCOMING,
-      user: event.tx.initiatedBy,
```

Comment:
isnt this necessary?

Reply:
Yes, its implemented for all earn events in earnBaseEventData now!